src/main/kotlin/no/nav/familie/ef/iverksett/vedtakstatistikk/VedtakstatistikkKafkaProducer.kt

```diff
@@ -1,21 +1,22 @@
 package no.nav.familie.ef.iverksett.vedtakstatistikk
 
+import no.nav.familie.eksterne.kontrakter.ef.BehandlingDVH
 import org.slf4j.LoggerFactory
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.kafka.core.KafkaTemplate
 import org.springframework.stereotype.Service
 
 @Service
 class VedtakstatistikkKafkaProducer(
-    private val kafkaTemplate: KafkaTemplate<String, String>
+    private val kafkaTemplate: KafkaTemplate<String, BehandlingDVH>
 ) {
     @Value("\${ENSLIG_FORSORGER_VEDTAK_TOPIC}")
     lateinit var topic: String
 
     private val logger = LoggerFactory.getLogger(javaClass)
     private val secureLogger = LoggerFactory.getLogger("secureLogger")
 
-    fun sendVedtak(vedtakStatistikk: String) {
+    fun sendVedtak(vedtakStatistikk: BehandlingDVH) {
```

Comment:
Klarer denne av å sende json uten att man har konfigurert opp noe? for kafkaproducern?

Reply:
Du har rett, det må i såfall konfigureres opp serialisering for producer. Tenker det er best å gjøre om objektet til Json-string når det sendes på topic